recipes/_packaging.rb

```diff
@@ -33,7 +33,11 @@
 elsif freebsd?
   package 'devel/ncurses'
 elsif rhel?
-  package 'fakeroot'
+  if node['platform_version'].satisfies?('>= 7') && (node["kernel"]["machine"] == "ppc64le" || node["kernel"]["machine"] == "ppc64")
```

Comment:
Let's be sure to take advantage of the new `ppc64?`, `ppc64le?` helpers you added to chef-sugar.